### PR TITLE
[KB-148] Thesis report refactor as per supervisor feedback 13/03

### PR DESCRIPTION
**Pull request by Pattrigue**

### CODE REVIEW

**Review by Ivikhostrup — APPROVED** [review]

File: report_thesis/src/sections/background.tex [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+Thanks to its sophisticated equipment, Curiosity was able to find evidence of past habitable environments on Mars based on chemical and mineral findings early in its mission~\cite{marsnasagov_msl}.
+
+One of the instruments aboard the rover is the \gls{chemcam} instrument, which is a remote-sensing laser instrument used to gather \gls{libs} data from geological samples on Mars.
+\gls{libs} is a technique that enables rapid analysis by using a laser to ablate and remove surface contaminants to expose the underlying material and generate a plasma plume from the now-exposed sample material~\cite{wiensChemcam2012}.
+This plasma plume emits light that is captured through three distinct spectrometers to collect a series of spectral readings.
+These spectra consist of emission lines that can be associated with the concentration of a specific element, and their intensity reflects the concentration of that element in the sample.
+Consequently, a spectra serves as a complex, multi-dimensional fingerprint of the elemental composition of the examined geological samples~\cite{cleggRecalibrationMarsScience2017}.
+
+\subsection{Baseline \& Replica}
```
Comment (chhoumann) [inline report_thesis/src/sections/background.tex:19]: Remove `baseline.tex`? Now that this section basically replaces it

File: report_thesis/src/sections/background.tex [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+We provide a detailed description of the \gls{moc} model in \citet{p9_paper}.
+
+\begin{figure}
+	\centering
+	\includegraphics[width=0.225\textwidth]{images/moc_pipeline.png}
+	\caption{Overview of the \gls{moc} model.}
+	\label{fig:moc_pipeline}
+\end{figure}
+
+In \citet{p9_paper}, we presented our efforts to replicate the \gls{moc} model.
```
Comment (chhoumann) [inline report_thesis/src/sections/background.tex:35]: image

The figure doesn't separate the text, so we're basically just repeating ourselves here

**Review by chhoumann — APPROVED** [review]

File: report_thesis/src/sections/background.tex (hunk 1 above)
Comment (Ivikhostrup) [reply report_thesis/src/sections/background.tex:19]: Resolved

File: report_thesis/src/sections/background.tex (hunk 2 above)
Comment (Ivikhostrup) [reply report_thesis/src/sections/background.tex]: Resolved

**Pull request merged**